projects/arlas-components/src/lib/components/results/result-list/result-list.component.scss

```diff
@@ -119,10 +119,26 @@
   color: #aaa;
 }
 
-tr:hover td.resultitem__cell.toggle .list_actions {
-  display: flex;
-  flex-direction: row-reverse;
-  align-items: center;
+tr {
+  &:hover {
+    .toggle {
+      .actions {
+        arlas-result-actions {
+          .list_actions {
+            .not_activated {
+              display: block;
+            }
+          }
+        }
+      }
+    }
+
+  }
+}
+
+
+tr:hover td.resultitem__cell.toggle ::ng-deep .list_active_actions {
```

Comment:
This could be integrated in the block above

Reply:
fixed